modules/nf-scil/segmentation/freesurferseg/tests/main.nf.test

```diff
@@ -0,0 +1,32 @@
+nextflow_process {
+
+    name "Test Process SEGMENTATION_FREESURFERSEG"
+    script "../main.nf"
+    process "SEGMENTATION_FREESURFERSEG"
+
```

Comment:
The old `nextflow.config` file had a `publishDir = {...}` parameter. Even though you don't need it now, you might want to add it if you switch to `load_test_data`.